vyxal/elements.py

```diff
@@ -3102,7 +4174,11 @@ def zero_slice(lhs, rhs, ctx):
     "T": process_element(truthy_indicies, 1),
     "U": process_element(uniquify, 1),
     "V": process_element(replace, 3),
-    "W": ("stack = [list(deep_copy(stack))]", 0),
+    "W": (
+        "temp = list(deep_copy(stack)); pop(stack, len(stack), ctx); "
+        "stack.append(temp)",
+        0,
```

Comment:
Because of scoping. Simply doing `stack = [stack]` was making the tests fail because a new local stack was being created. It appears to be a necessary evil as long as we make `stack` a global variable like that.

Reply:
Ah ok, that makes sense. What I was mainly commenting on was actually the `deep_copy` - I understand it needs to be copied once (otherwise the stack would contain a circular reference to itself), but why a deep copy?.

By the way, this can be, more idiomatically:

```
stack[:] = [stack.copy()]
# or deep_copy(stack)
```

Reply:
No real reason for a deep copy, honestly, I was just being unpythonic.